lms/djangoapps/discussion/static/discussion/js/views/discussion_user_profile_view.js

```diff
@@ -48,7 +48,7 @@
                         return view;
                     });
                     pageUrlFunc = function(page) {
-                        baseUri.clone().addSearch('page', page);
+                        return baseUri.clone().addSearch('page', page).toString();
```

Comment:
It's a [URI object](https://medialize.github.io/URI.js/docs.html). Not sure why I had to add toString here, but trying to call `window.loaction.push` on a straight URI object (rather than calling toString first) was the reason pagination was broken.